Tests/APITesters/ObjCAPITester/ObjCAPITester/RCPurchasesAPI.m

```diff
@@ -179,10 +173,6 @@ + (void)checkAPI {
     [p logIn:@"" completion:^(RCCustomerInfo *i, BOOL created, NSError *e) { }];
     [p logOutWithCompletion:^(RCCustomerInfo *i, NSError *e) { }];
 
-    #if ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
-    [p switchUserToNewAppUserID:@""];
-    #endif
```

Comment:
I noticed we don't have API tests for ObjC for the new mode... But not sure how necessary that is.